### PR TITLE
Add hooks repo

### PR DESCRIPTION
**Pull request by tomislav-s**
This is the initial setup for React specific packages, and the React hooks implementation as well.

I accidentally already pushed some code to `main` when i was setting up the repo (sry) - so feel free to comment any of the code that's already on `main` 🙂 .

### CODE REVIEW

**Review by heldinz — COMMENTED** [review]
Looks like a great start, thank you! Just wondering why we have `sort-keys` off in `.eslintrc`?

We probably need to talk about how we use Prettier in our projects, btw: we don't have a standard config in a repo anywhere since it's just done with a `.prettierrc` file, but I think we should standardize it. (Will add it to Stammtisch for this afternoon.)

Could we please also get a README for the monorepo?

**Comment by tomislav-s** [comment]
> Looks like a great start, thank you! Just wondering why we have `sort-keys` off in `.eslintrc`?

I copied that setup from our tools and config monorepo. Should I remove it?

> We probably need to talk about how we use Prettier in our projects, btw: we don't have a standard config in a repo anywhere since it's just done with a `.prettierrc` file, but I think we should standardize it. (Will add it to Stammtisch for this afternoon.)

Thanks 🙂 

> Could we please also get a README for the monorepo?

It should be present in this PR - https://github.com/acolorbright/react-tools/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

It's mostly copied from our other monorepo with minor differences (like description).

P.S. just realised that I should add independent mode here as well, will do this now

File: packages/react-hooks/package.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
 {
   "name": "@acolorbright/react-hooks",
   "version": "0.0.0",
-  "description": "ACB's package for reausable react hooks",
+  "description": "ACB's package for reausable React hooks",
```
Comment (heldinz) [inline packages/react-hooks/package.json]: Tiny typo: s/reausable/reusable/

**Review by heldinz — DISMISSED** [review]
Nice!

**Review by heldinz — APPROVED** [review]
Looks like a great start to me, thanks!

**Pull request merged**